cmd/nvidia-cdi-hook/main.go

```diff
@@ -72,6 +73,35 @@ func main() {
 			commands.IssueUnsupportedHookWarning(logger, cmd)
 			return nil
 		},
+		// Handle usage errors to catch unrecognized hooks
+		OnUsageError: func(ctx context.Context, cmd *cli.Command, err error, isSubcommand bool) error {
+			errMsg := err.Error()
+
+			// Check if this is a "No help topic for" error (unrecognized
```

Comment:
Since in Urfave we have a [`CommandNotFound` handler](https://github.com/urfave/cli/blob/de9b5b4e61b0cce132f2a2bc73261eb0601d76ce/help.go#L318-L329) does it not make sense to handle this separately?